README.md

```diff
@@ -95,11 +105,26 @@ During its execution, the daemon PID will be stored in:
 
 ### Logging
 
-By default, log messages will be writted to the pxp-agent.log file in:
- - \*nix: */var/log/puppetlabs/pxp-agent*
- - Windows: *C:\ProgramData\PuppetLabs\pxp-agent\var\log*
+## *nix
+
+By default, log messages will be written to
+*/var/log/puppetlabs/pxp-agent/pxp-agent.log*.
+
+You can specify a different file with the `--logfile` option.
+
+When running in foreground mode, it is possible to display log messages on
+console by using an hyphen instead of a file path: `--logfile -`.
+
+## Windows
 
-You can specify a different directory with the `--logdir` flag.
+By default, log messages are written to the stdout stream.
+
+You can specify a log file with `--logfile` option. In that case, no
+file rotation will take place.
+
+Note that it is not possible to have log messages displayed only on console.
```

Comment:
That's wrong. I forgot to remove it after copying from https://gist.github.com/parisiale/e9ba57b2062cb5eaeed0, which was meant for PCP-100 discussion.

Sorry for that.